circle.yml

```diff
@@ -11,9 +11,10 @@ machine:
 dependencies:
   pre:
     - sudo pip install docker-compose
+    - sudo cp /usr/bin/docker /usr/local/bin/docker
```

Comment:
Main reason was to keep code symmetric -- `DockerComposeExecutable` had its locations as `System.getenv("DOCKER_COMPOSE_LOCATION")` and `/usr/local/bin/docker-compose`, and it works because the "pip install" installs the executable there. Agree this is a bit awkward, though -- I'll just add `/user/bin/<exec>` as a third location for both in the updated PR and remove the extra line from circle.yml